doc/source/ray-air/examples/dolly_lightning_fsdp_finetuning.ipynb

```diff
@@ -76,7 +76,7 @@
        "    </tr>\n",
        "    <tr>\n",
        "        <td style=\"text-align: left\"><b>Ray version:</b></td>\n",
-       "        <td style=\"text-align: left\"><b>3.0.0.dev0</b></td>\n",
+       "        <td style=\"text-align: left\"><b>2.7.0rc0</b></td>\n",
```

Comment:
why is this one changed from `3.0.0.dev0` shouldn't this be `2.7.0` on the 2.7.0 release branch

Reply:
I guess this is cherrypicked in but with the wrong version number.

@matthewdeng or @GeneDer , could you fix this on the 2.7.0 branch?

Reply:
makes sense, let me push a PR on 2.7.0 real quick

Reply:
We should probably just delete the output blocks instead. cc @woshiyyya

Reply:
oh if that's the case I will let you guys figure it out XP. There are a few .ipynb files that need to clean up here